galaxy/templates/configmap-nginx.yaml

```
@@ -69,6 +69,19 @@ data:
                 add_header X-Content-Type-Options nosniff;
             }
 
+            location {{ template "galaxy.add_trailing_slash" .Values.ingress.path }}api/upload/resumable_upload {
+                # Disable request and response buffering
+                proxy_request_buffering  off;
+                proxy_buffering          off;
+                proxy_http_version       1.1;
+                # Add X-Forwarded-* headers
+                proxy_set_header X-Forwarded-Host $host;
+                proxy_set_header X-Forwarded-Proto $scheme;
+                proxy_set_header         Upgrade $http_upgrade;
+                proxy_set_header         Connection "upgrade";
+                client_max_body_size     {{ .Values.nginx.conf.client_max_body_size }};
+                proxy_pass http://{{ template "galaxy.fullname" . }}-tusd:1080/files/;
```

```suggestion
                proxy_pass http://{{ template "galaxy.fullname" . }}-tusd:1080/files;
```
I also ran into the same thing and then didn't update the docs ...